### PR TITLE
[AUD-110] Fix Unfollow from Desktop User List

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

- Changes follow button to immediately show 'follow' after unfollowing, as per discussion w product and design
- Fixes unfollow from userlist broken on desktop
- Looks like we were missing unsubscribe when unfollow on a few buttons, following the behavior on mobile here. Not 100% sure this is necessary?

### Dragons

None

### How Has This Been Tested?

Locally

### CODE REVIEW

**Comment by raymondjacobson** [comment]
Seems reasonable, can you double check w/ forrest on this copy change tho? I think the follow/unfollow stuff was intentional ( but maybe confusing )

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/aud-110

**Comment by piazzatron** [comment]
> Seems reasonable, can you double check w/ forrest on this copy change tho? I think the follow/unfollow stuff was intentional ( but maybe confusing )

This ended up being really funny. 

A) This wasn't even the bug QA reported
but
B) Forrest agreed that the current button state was real wonky.

Ended up going back and forth w him and Julian a bit, ended up with this.

File: src/components/artist/ArtistPopover.tsx [hunk 1]
```diff
@@ -154,8 +155,10 @@ const mapDispatchToProps = (dispatch: Dispatch) => ({
   goToRoute: (route: string) => dispatch(pushRoute(route)),
   onFollow: (userId: ID) =>
     dispatch(socialActions.followUser(userId, FollowSource.HOVER_TILE)),
-  onUnfollow: (userId: ID) =>
+  onUnfollow: (userId: ID) => {
     dispatch(socialActions.unfollowUser(userId, FollowSource.HOVER_TILE))
+    dispatch(setNotificationSubscription(userId, false, true))
```
Comment (raymondjacobson) [inline src/components/artist/ArtistPopover.tsx:160]: +1 ty for catching these

**Review by raymondjacobson — APPROVED** [review]

**Pull request merged**